AI/AIInterface.h

```diff
@@ -345,6 +345,9 @@ namespace AIInterface {
     int IssueChangeProductionQuantityOrder(int queue_index, int new_quantity, int new_blocksize);
     int IssueRequeueProductionOrder(int old_queue_index, int new_queue_index);
     int IssueDequeueProductionOrder(int queue_index);
+    int IssueChangeProductionQuantityOrder(int queue_index, int new_quantity, int new_blocksize);
```

Comment:
?

Reply:
that's a whoops! copy-paste error. will be fixed